cloudwatch-firehose-es/src/run.py

```diff
@@ -111,11 +111,46 @@ def processRecords(records):
                 if i == 0:
                     cw_log_data = transformLogEvent(event, cloudwatch_info)
 
-                    yield {
-                        'data' : str(base64.b64encode(cw_log_data.encode()), 'utf-8'),
-                        'result' : 'Ok',
-                        'recordId' : recId
-                    }
+                    # Determine if extra ways to categorize the event were provided
```

Comment:
It might be safer here to add this as an optional feature that can be enabled with an env var 

So something like 

```
if os.environ.get("LOG_CATEGORIES", None) is not None: 
```

and at the top just need to import os 
```
import os 
```